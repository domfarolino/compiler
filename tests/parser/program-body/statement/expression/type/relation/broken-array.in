program dominic is

  bool result[0: 9];
  bool left[0: 9];

begin

  result := left == false;

end program.
