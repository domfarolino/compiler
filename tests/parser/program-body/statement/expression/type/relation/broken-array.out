[1;31mLine 8: [Relation]: The relational operators are not implemented for arrays at this time
Line 8: Error parsing program body
[0m